tracer/src/Datadog.AutoInstrumentation.NativeLoader/dynamic_dispatcher.h

```diff
@@ -3,6 +3,7 @@
 #include <corprof.h>
 #include <functional>
 #include <vector>
+#include <memory>
```

Comment:
Why? Is it for linux build?

Reply:
Yes, under Linux I get this error when compiling without it:
```
   /home/robert/code/dd-trace-dotnet/tracer/src/Datadog.AutoInstrumentation.NativeLoader/dynamic_dispatcher.h:33:14: error: no template named 'unique_ptr' in namespace 'std'
           std::unique_ptr<IDynamicInstance> m_continuousProfilerInstance;
           ~~~~~^
```

I don't claim this is the best way to fix it, but it seems to work okay so far.

Reply:
Oh ok, yeah maybe there’s a preprocessor conditional somewhere for windows. But yeah the memory header is required for the smart pointer.